docs/kubernetes.md

```diff
@@ -11,14 +11,17 @@ This document explains how to configure SOCI on Kubernetes. For a hands on examp
 ## Configuration
 
 SOCI on kubernetes requires two pieces of configuration:
+
 1) [Containerd Configuration](#containerd-configuration) to launch containers with SOCI
 2) [Registry Authentication Configuration](#registry-authentication-configuration) so that SOCI can pull images from non-public container registries
 
 ### Containerd configuration
 
-To configure containerd to launch containers with SOCI, add the following snippet to the containerd config. The config is located at `/etc/containerd/config.toml` by default. 
+To configure containerd to launch containers with SOCI, add the following snippet to the containerd config. The config is located at `/etc/containerd/config.toml` by default.
 
-```
+#### containerd pre-2.0 configuration
```

Comment:
nit: can we do `containerd 1.x configuration` and `containerd 2.x configuration`?